### PR TITLE
Improve the `tap-highlight-color`

### PR DESCRIPTION
**Pull request by abdonrd**
Like the paper-tabs: https://github.com/PolymerElements/paper-tabs/blob/cf354bb2df0b8ec8524bde499560197e27dc7034/paper-tabs.html#L112-L114

Reference: https://github.com/PolymerElements/paper-fab/pull/48/files#r50937454

### CODE REVIEW

**Comment by abdonrd** [comment]
Friendly ping to @bicknellr.

File: paper-icon-button.html [hunk 1]
```diff
@@ -76,7 +76,11 @@
         position: relative;
         padding: 8px;
         outline: none;
-        -webkit-tap-highlight-color: rgba(0,0,0,0);
+
+        /* NOTE: Both values are needed, since some phones require the value to be `transparent`. */
```
Comment (bicknellr) [inline paper-icon-button.html]: Is there anything that supports `rgba(...)` but not `transparent`? If not, we should use `transparent` only.

Comment (abdonrd) [inline paper-icon-button.html]: @bicknellr Yes, the reference: https://github.com/PolymerElements/paper-fab/pull/48/files#r50937454 :)

Comment (bicknellr) [inline paper-icon-button.html]: Ok - I need to update this change in paper-checkbox too then ([here's a PR I merged yesterday](https://github.com/PolymerElements/paper-checkbox/pull/127)).

Comment (abdonrd) [inline paper-icon-button.html]: Nice! :)

**Comment by bicknellr** [comment]
Oh, could you get this one also? https://github.com/PolymerElements/paper-icon-button/blob/99395773ef073a20dab1c4bb3e6230c59a42f572/paper-icon-button-light.html#L35

File: paper-icon-button.html (hunk 1 above)
Comment (bicknellr) [inline paper-icon-button.html]: Wait... is `rgba(0, 0, 0, 0)` alone sufficient? They mentioned that the color `transparent` wasn't supported on certain versions of iOS, but is there anywhere that `rgba(0, 0, 0, 0)` isn't supported?

**Comment by abdonrd** [comment]
Done! https://github.com/PolymerElements/paper-icon-button/pull/88

File: paper-icon-button.html (hunk 1 above)
Comment (abdonrd) [inline paper-icon-button.html]: Sincerely, I'm not sure... In a quick search: http://phonegap-tips.com/articles/essential-phonegap-css-webkit-tap-highlight-color.html#comment-904717757

Comment (bicknellr) [inline paper-icon-button.html]: Given that we have precedent for including both, I'll go with it. Maybe later we'll discover we can remove one.

**Comment by bicknellr** [comment]
Closing since your new PR has both fixes.

**Pull request closed without merging**